src/js/components/scene-editor/components/audio-media-preview-player.jsx

```diff
@@ -0,0 +1,49 @@
+'use strict';
+var React = require('react');
+var _ = require('lodash');
+var soundCloud = require('../../../utils/sound-cloud');
+
+var AudioMediaPreviewPlayer = React.createClass({
```

Comment:
Refactored the Audio Preview code into its own class.  Makes more sense this way and more reusable